specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/stable/2019-09-01/QueryPacks.json

```diff
@@ -222,14 +222,26 @@
             "schema": {
               "$ref": "#/definitions/LogAnalyticsQueryPack"
             }
+          },
```

Comment:
Does it need to remove the block of "200"?

Reply:
it can return 200 if a name is passed, whenever a name is not passed 201 is returned

Reply:
@dabenhamMic , the symptom is that using [QueryPacks_CreateOrUpdate](https://github.com/Azure/azure-rest-api-specs/blob/db6d33733cd1eb939b863a6cdbcb9de12ac002e1/specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/stable/2019-09-01/QueryPacks.json#L235) API to create the resource failed because API always returns 201 but 201 isn't defined in the [response section](https://github.com/Azure/azure-rest-api-specs/blob/db6d33733cd1eb939b863a6cdbcb9de12ac002e1/specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/stable/2019-09-01/QueryPacks.json#L266)

And I found it always returns 201 even if a name is passed after tested. Could you double check the API behavior since it's different with what you mentioned above?

Request Payload:
```
PUT https://management.azure.com/subscriptions/xx-xx-xx-xx/resourceGroups/acctestRG-LA/providers/Microsoft.OperationalInsights/queryPacks/acctestlaqp

{
    "location": "westus",
    "properties": {},
    "tags": null
}
```

API Response:
![image](https://user-images.githubusercontent.com/19754191/189886075-4e07c660-216f-45b3-9803-4e365c8d5818.png)